widgetssdk/src/main/java/com/glia/widgets/core/engagement/GliaEngagementConfigRepository.kt

```diff
@@ -0,0 +1,13 @@
+package com.glia.widgets.core.engagement
+
+import com.glia.widgets.chat.ChatType
+
+class GliaEngagementConfigRepository {
+    var queueIds = emptyArray<String>()
+    var chatType = ChatType.LIVE_CHAT
+
+    fun clear() {
```

Comment:
maybe reset() could be a better name for this function?

Reply:
You're right!